freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -107,6 +109,16 @@ public SyncOperationStatus getSyncOperationStatus(@NotNull String operationId) {
                 operationService.getOperationForAccountIdAndOperationId(accountId, operationId));
     }
 
+    private String determineAccountId(String callingActor, String requestedAccountId) {
+        if (requestedAccountId == null) {
+            return ThreadBasedUserCrnProvider.getAccountId();
+        }
+        if (!InternalCrnBuilder.isInternalCrn(callingActor)) {
```

Comment:
you should also allow setting the account id to your own account

Reply:
Fixed.